Scripts/Source/minai_AIFF.psc

```diff
@@ -80,9 +88,40 @@ Function Maintenance(minai_MainQuestController _main)
   InitializeActionRegistry()
   ; Test, remove this later
   ; StoreContext("minai", "testKey", "This is dynamically persisted context!", 1200)
+
+  InitFollow()
 EndFunction
 
+Function StartFollowTarget(actor akNpc, actor akTarget)
```

Comment:
We should have a check (Probably on enabling the action to begin with) that the NPC isn't in a scene, as this will for sure break scripted events. :D

Reply:
arh.. that's a good idea.

Reply:
Might as well stop following before any scene start.